sdk/agrifood/azure-verticals-agrifood-farming/README.md

```diff
@@ -166,7 +166,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 [product_documentation]: https://aka.ms/FarmBeatsProductDocumentationPaaS
 [azure_portal]: https://portal.azure.com
 [jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
-[package]: https://mvnrepository.com/artifact/com.azure/azure-verticals-agrifood-farming
+[package]: https://central.sonatype.com/search?namespace=com.azure&name=azure-verticals-agrifood-farming
```

Comment:
We should use the overview link instead of the `search` link.

This link takes directly to the latest version of the library.
https://central.sonatype.com/artifact/com.azure/azure-verticals-agrifood-farming

Reply:
Great catch.  Let's use this instead

Reply:
Updated to use the better URL pattern.